docs/sphinx/_toc.yml.in

```diff
@@ -37,21 +37,48 @@ subtrees:
 
 - caption: Reference
   entries:
-  - file: doxygen/html/index
+  - file: reference/hip_runtime_api_reference
+    subtrees:
+    - entries:
+      - file: reference/hip_runtime_api/initialization_and_version
+      - file: reference/hip_runtime_api/device_management
+      - file: reference/hip_runtime_api/execution_control
+      - file: reference/hip_runtime_api/error_handling
+      - file: reference/hip_runtime_api/stream_management
+      - file: reference/hip_runtime_api/stream_memory_operations
+      - file: reference/hip_runtime_api/event_management
+      - file: reference/hip_runtime_api/memory_management
+        subtrees:
+        - entries:
+          - file: reference/hip_runtime_api/memory_management/external_resource_interoperability
+          - file: reference/hip_runtime_api/memory_management/stream_ordered_memory_allocator
+          - file: reference/hip_runtime_api/memory_management/unified_memory_reference
+          - file: reference/hip_runtime_api/memory_management/virtual_memory_reference
+          - file: reference/hip_runtime_api/memory_management/texture_management
+          - file: reference/hip_runtime_api/memory_management/texture_management_deprecated
+          - file: reference/hip_runtime_api/memory_management/surface_object
+      - file: reference/hip_runtime_api/peer_to_peer_device_memory_access
+      - file: reference/hip_runtime_api/context_management
+      - file: reference/hip_runtime_api/module_management
+      - file: reference/hip_runtime_api/occupancy
+      - file: reference/hip_runtime_api/profiler_control
+      - file: reference/hip_runtime_api/launch_api
+      - file: reference/hip_runtime_api/runtime_compilation
+      - file: reference/hip_runtime_api/callback_activity_apis
+      - file: reference/hip_runtime_api/graph_management
+      - file: reference/hip_runtime_api/opengl_interoperability
+      - file: reference/hip_runtime_api/cooperative_groups_reference
+      - file: reference/hip_runtime_api/global_enum_and_defines
```

Comment:
I would enter the Global defines, enums, structs and files submodule here, and then have the subtopics.

Reply:
Updated.